src/style.scss

```diff
@@ -1,12 +1,14 @@
-// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
-// See LICENSE in the project root for license information.
+/* Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license */
+/* See LICENSE in the project root for license information. */
 
-// ---------------------------------------------------------------------
+/* --------------------------------------------------------------------- */
 
-// Import Auro Sass variables
+/* stylelint-disable at-rule-no-unknown */
```

Comment:
This rule was disabled because there was an error in the console saying that the file was using `@` rules that were unknown to the linter. These `@` rules being used are common selectors used across multiple components, so I decided to disable the rule as we need these selectors.

Reply:
Instead of disabling the rule we might be able to keep some linting functionality by [adding the unknown `@` rules to the config so they are no longer unknown](https://stylelint.io/user-guide/rules/at-rule-no-unknown/#optional-secondary-options)